shared/js/background/before-request.es6.js

```diff
@@ -321,6 +346,10 @@ function blockHandleResponse (thisTab, requestData) {
             // @ts-ignore
             Companies.add(tracker.tracker.owner)
 
+            // publish the parent's display name only
+            const displayName = utils.findParentDisplayName(requestData.url)
+            emitter.emit(TrackerBlockedEvent.eventName, new TrackerBlockedEvent({ companyDisplayName: displayName }))
```

Comment:
Should we skip sending stats for incognito pages maybe?

Reply:
Sounds good, do we have any examples of doing that elsewhere?

Reply:
You can use the `incognito` property on `requestData` to check if the request is from a private tab. (When looking for a docs reference, it seems that this attributed is not in the Chrome docs, but still on MDN, so maybe check that this still exists https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/onBeforeRequest).

Reply:
From what I can see, our handlers do not see any requests from incognito tabs anyway? Unless there's some special way to test this that you know of?

Reply:
Yeah it seems like our extension cannot be enabled in incognito mode anyway, so this is probably not an issue?

Reply:
You have to manually allow the extension in incognito mode:
 1. From `chrome://extensions/`, click 'details' for the extension. ![Screenshot 2023-02-01 at 13 26 24](https://user-images.githubusercontent.com/248111/216042389-d234017f-887f-4d8b-bbe5-872cd2aeeaf2.png)
 2. Toggle 'Allow in Incognito' on.

![Screenshot 2023-02-01 at 13 26 28](https://user-images.githubusercontent.com/248111/216042398-7ce45073-0ad4-4ef2-9925-fa8f0bb5bd16.png)

Reply:
@sammacbeth - thanks for this

I looked into it, you can no longer detect that a request came from an incognito window - the property shown in those MDN docs is absent on all requests.

It seems like this is by design and I don't think we can/should pursue solving it right now :)